prototyping/src/clibrary.cpp

```diff
@@ -0,0 +1,26 @@
+//#include <QVector>
```

Comment:
This line produces the following error:

`src.compiler.CompilationError: /Users/guadaluperomero/amarex/amarex-webtool/prototyping/src/clibrary.cpp:1:10: fatal error: 'QVector' file not found`

I think the problem is that I need to set the path to the header file for QVector, like we do [here](https://github.com/technologiestiftung/amarex-webtool) at step 3 of the compilation. But in this prototype, we are using GCC instead of qmake for the compilation.